assemblies/assembly-enabling-authentication.adoc

```diff
@@ -34,7 +34,7 @@ For each provider that you want to use, follow the dedicated procedure to comple
 . Set up the shared secret that the authentication provider and {product} require to communicate.
```

Comment:
I noticed that some of the assemblies contain these anchor IDs:
`:_mod-docs-content-type: ASSEMBLY`
but some don't. The modular guide says it is optional if you have `assembly-` in the name of the file, but it should be uniform across the project. I would recommend unifying the assemblies and either include the anchor ID or not (including is probably the sfer option).

Reply:
Sure, @eromanova97 ! I'll let the team know and see if we can work on this!